src/functions-and-arrays.js

```diff
@@ -1,41 +1,96 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) { return num1 }
+  else { return num2 }
+}
+result = maxOfTwoNumbers(3, 5);
+console.log(result);
+
 
 
 
 // Iteration #2: Find longest word
+
+function findLongestWord(words) {
+
+  let wordLength = [];
+  wordLength.push(words[0]);
+  //console.log(wordLength);
+  //console.log(typeof (wordLength));
+
+  for (let i = 0; i < words.length; i++) {
+    if (wordLength.length < words[i].length) {
```

Comment:
Better store the longest word in a string, not an array and just compare the length by using longestWord.length